packages/control/ev.py

```diff
@@ -116,8 +116,7 @@ def et_factory() -> Et:
 
 @dataclass
 class ChargeTemplateData:
-    name: str = "neues Lade-Profil"
-    disable_after_unplug: bool = False
+    name: str = "Standard-Lade-Profil"
```

Comment:
```suggestion
    name: str = "neues Lade-Profil"
```
Hier ist was beim Rebasen schief gelaufen.

Reply:
Sorry. Habs committed. Ja beim Auflösen des Mergekonflikts